lua/ttt2/libraries/targetid.lua

```diff
@@ -0,0 +1,546 @@
+---
+-- @module TargetID
+-- @desc Handling operations to display the TargetID
+-- @author Mineotopia
+
+if SERVER then
+    AddCSLuaFile()
+    return
+end
+
+-- Global to local variables
+local ParT = LANG.GetParamTranslation
+local TryT = LANG.TryTranslation
+
+-- Variables for calculations
+local MAX_TRACE_LENGTH = math.sqrt(3) * 32768
+
+-- Convars for targetid
+local cvDeteOnlyConfirm = GetConVar("ttt2_confirm_detective_only")
+local cvDeteOnlyInspect = GetConVar("ttt2_inspect_detective_only")
+
+-- Materials for targetid
+local materialTButton = Material("vgui/ttt/tid/tid_big_tbutton_pointer")
+local materialRing = Material("effects/select_ring")
+local materialRoleUnknown = Material("vgui/ttt/tid/tid_big_role_not_known")
+local materialDisguised = Material("vgui/ttt/perks/hud_disguiser.png")
+local materialCorpse = Material("vgui/ttt/tid/tid_big_corpse")
+local materialCredits = Material("vgui/ttt/tid/tid_credits")
+local materialDetective = Material("vgui/ttt/tid/tid_detective")
+local materialLocked = Material("vgui/ttt/tid/tid_locked")
+local materialAutoClose = Material("vgui/ttt/tid/tid_auto_close")
+local materialDoor = Material("vgui/ttt/tid/tid_big_door")
+local materialDestructible = Material("vgui/ttt/tid/tid_destructible")
+local materialDNATargetID = Material("vgui/ttt/dnascanner/dna_hud")
+
+targetid = targetid or {}
+
+---
+-- This function handles finding Entities by casting a ray from a point in a direction, filtering out certain entities
+-- Use this in combination with the hook @GM:TTTModifyTargetedEntity to create your own Remote Camera with TargetIDs.
+-- e.g. This is used in @GM:HUDDrawTargetID before drawing the TargetIDs. Use that code as example.
+-- @note This finds the next Entity, that doesn't get filtered out and can get hit by a bullet, from a position in a direction.
+-- @param vector pos Position of Ray Origin.
+-- @param vector dir Direction of the Ray. Should be normalized.
+-- @param table filter List of all @{Entity}s that should be filtered out.
+-- @return entity The Entity that got found
+-- @return number The Distance between the Origin and the Entity
+-- @realm client
+function targetid.FindEntityAlongView(pos, dir, filter)
```

Comment:
So I initialized targetid correctly and yesterday it worked like this.
Now I'm getting this error after I moved more functions into targetid.
But the linter doesnt point to anything, so I'm confused
![grafik](https://user-images.githubusercontent.com/72046466/96757381-a5780480-13d5-11eb-93e1-de681784c82f.png)

Reply:
The error seems to be in the variables I located on top. Outcommenting them allows the targetid to be created normally.
Narrowed it down to LANG. and Key() which seem to stop it.